commit 52608ba20546139dc76cca8a46c1d901455d5450
Author: Niklas Söderlund <niklas.soderlund@ericsson.com>
Date:   Wed Aug 8 12:30:56 2012 -0300

    i5100_edac: probe for device 19 function 0
    
    Probe and store the device handle for the device 19 function 0 during
    driver initialization. The device is used during fault injection.
    
    Signed-off-by: Niklas Söderlund <niklas.soderlund@ericsson.com>
    Signed-off-by: Mauro Carvalho Chehab <mchehab@redhat.com>

diff --git a/include/linux/pci_ids.h b/include/linux/pci_ids.h
index 0eb65796bcb9..d0d1e801e350 100644
--- a/include/linux/pci_ids.h
+++ b/include/linux/pci_ids.h
@@ -2776,6 +2776,7 @@
 #define PCI_DEVICE_ID_INTEL_JAKETOWN_UBOX	0x3ce0
 #define PCI_DEVICE_ID_INTEL_IOAT_SNB	0x402f
 #define PCI_DEVICE_ID_INTEL_5100_16	0x65f0
+#define PCI_DEVICE_ID_INTEL_5100_19	0x65f3
 #define PCI_DEVICE_ID_INTEL_5100_21	0x65f5
 #define PCI_DEVICE_ID_INTEL_5100_22	0x65f6
 #define PCI_DEVICE_ID_INTEL_5400_ERR	0x4030